cmd/serv.go

```diff
@@ -243,7 +243,7 @@ func runServ(c *cli.Context) error {
 	os.Setenv(models.EnvPusherID, strconv.FormatInt(results.UserID, 10))
 	os.Setenv(models.EnvRepoID, strconv.FormatInt(results.RepoID, 10))
 	os.Setenv(models.EnvPRID, fmt.Sprintf("%d", 0))
-	os.Setenv(models.EnvIsDeployKey, fmt.Sprintf("%t", results.IsDeployKey))
+	os.Setenv(models.EnvDeployKeyID, fmt.Sprintf("%d", results.DeployKeyID))
 	os.Setenv(models.EnvKeyID, fmt.Sprintf("%d", results.KeyID))
```

Comment:
Isn't `EnvKeyID` the deploy key id?

Reply:
The old code is messy. I had updated the comments in this PR.


![image](https://user-images.githubusercontent.com/2114189/156936949-f76dd066-eb11-4cf6-91f4-155f94117326.png)

![image](https://user-images.githubusercontent.com/2114189/156936979-71ef62ed-8cd7-4ee3-9007-19819de17218.png)

![image](https://user-images.githubusercontent.com/2114189/156937005-34b52826-25cf-4824-b988-d42802816039.png)